ui/src/main/scripts/plugins/e2e/html/e2e.html

```diff
@@ -4,7 +4,7 @@
     <div class="card-pf-body">
       <form class="form-horizontal form-inline row">
         <div class="form-group col-md-5">
-          <label class="control-label col-md-3" for="rootnode">Root Node </label>
+          <label class="control-label col-md-3" for="rootnode">Endpoint </label>
```

Comment:
Would prefer "Initial Endpoint"?  - root doesn't really have any meaning to the end user.